ko/Messages.json

```diff
@@ -201,50 +224,158 @@
+    "ACHIEVEMENT_DESCRIPTION_SKILL_FOREVER": "연속 10번 승리하세요",
+    "ACHIEVEMENT_NAME_WE_NEED_TO_GO_DEEPER": "심해 속으로",
+    "ACHIEVEMENT_DESCRIPTION_WE_NEED_TO_GO_DEEPER": "점수를 1800점 이하로 떨어트리세요",
+    "ACHIEVEMENT_NAME_GLADIATOR": "검투사",
+    "ACHIEVEMENT_DESCRIPTION_GLADIATOR": "50명의 다른 플레이어와 대결하세요",
+    "ACHIEVEMENT_NAME_TRAVELLER": "여행자",
+    "ACHIEVEMENT_DESCRIPTION_TRAVELLER": "화물 컨베이어로 당신의 유닛을 500 블록만큼 이동시키세요",
+    "ACHIEVEMENT_NAME_THIN_ICE": "살얼음판",
+    "ACHIEVEMENT_DESCRIPTION_THIN_ICE": "체력 10 이하에서부터 코어를 완전히 수리하세요",
+    "ACHIEVEMENT_NAME_COMEBACK": "개같이 부활",
+    "ACHIEVEMENT_DESCRIPTION_COMEBACK": "상대보다 코어를 4개 이상 적게 가지고 승리하세요",
+    "ACHIEVEMENT_NAME_DAVID_AND_GOLIATH": "다윗과 골리앗",
+    "ACHIEVEMENT_DESCRIPTION_DAVID_AND_GOLIATH": "적보다 낮은 단계의 유닛만을 사용해 승리하세요",
+    "ACHIEVEMENT_NAME_HARDWORKING": "근면성실",
+    "ACHIEVEMENT_DESCRIPTION_HARDWORKING": "3분 동안 쉬지 않고 채굴하세요",
+    "ACHIEVEMENT_NAME_I_VE_GOT_NO_POWA": "나 자신없어!",
+    "ACHIEVEMENT_DESCRIPTION_I_VE_GOT_NO_POWA": "발전기 없이 전력을 사용하는 포탑을 건설하세요",
+    "ACHIEVEMENT_NAME_PAINTER": "화가",
+    "ACHIEVEMENT_DESCRIPTION_PAINTER": "대결 중 픽셀아트를 건설하세요",
+    "ACHIEVEMENT_NAME_GETTING_STARTED": "시작하기",
+    "ACHIEVEMENT_DESCRIPTION_GETTING_STARTED": "튜토리얼을 마치세요",
+    "ACHIEVEMENT_NAME_ELEET_SECRET": "Eleet Secret",
+    "ACHIEVEMENT_DESCRIPTION_ELEET_SECRET": "비밀 업적",
+    "ACHIEVEMENT_NAME_PIECE_OF_A_CAKE": "케이크 한 조각",
+    "ACHIEVEMENT_DESCRIPTION_PIECE_OF_A_CAKE": "서버의 생일을 축하하는 동안 대결을 진행하세요",
+    "BALANCE_UNITS": "[yellow]모든 유닛 공통[]\n- 각 유닛의 [green]최대[]사거리에서 발사합니다\n- 유닛의 체력이 일정 비율 이하로 떨어질 경우, 가까운 수리 지점\uF848으로 [green]자동 귀환[]합니다 \n 기준: T1-70%, T2-60%, T3-50%, T4-40%, T5-30%\n\n[yellow]지상 유닛 공통[]\n- 대상보다 자신의 사거리가 길 경우, 적을 쏠 때 움직이지 않습니다\n- 이 기능은 메이스\uF7FF, 펄서\uF7FC, 퀘이사\uF7FB, 아트락스\uF7F9, 스피록트\uF7F8에게는 적용되지 않습니다 \n- 노바 계열 유닛은 깊은 물에 빠지지 않고 비행하여 건너갑니다\n\n[yellow]비행 유닛 공통[]\n- 공격 대상을 상대로 빙빙 돌지 않습니다\n- 중립 구조물을 무시합니다\n- 다양한 폭발(폭발성)은 66% 감소된 피해를 입힙니다\n\n[yellow]모노[]\uF7F1[yellow], 폴리[]\uF7F0[yellow] 그리고 메가[]\uF7EF\n- 기본적으로 주어진 작업을 수행하지 않을 때, 채굴합니다\n- 만약 구리\uF838와 납\uF837 둘 다 4000 이상이면, 상황에 따라 모래\uF834, 티타늄\uF832, 그리고 토륨\uF831을 채굴합니다\n- 자원이 충분하지 않을 때, 폴리\uF7F0는 더이상 밀린 재건 계획을 수행하지 않습니다.",
+    "BALANCE_STATS": "[yellow]능력치[]\n체력 변경점은 데미지 배수를 설정된 데미지 배수를 거치고 적용됩니다.  (로직 역호환 가능)\n\n대거\uF800: 재장전 14⇨13\n메이스\uF7FF: [red]♥[]500⇨650, 30⇨34, 재장전 14⇨11, [yellow]\uE86B[]4⇨7\n포트리스\uF7FE: [red]♥[]800⇨1000, 54⇨62\n노바\uF7FD: [red]♥[]120⇨160, 14⇨16\n퀘이사\uF7FB: [red]♥[]650⇨700\n크롤러\uF7FA: [red]♥[]180⇨220, 85⇨90, [green]⛷[]1⇨1.05\nAtrax\uF7F9: reload 10⇨9, [green]⛷[]0.5⇨0.8\n스피록트\uF7F8: 재장전 15⇨14, [green]⛷[]0.4⇨0.7\n아키드\uF7F8: [green]\u26F7[]0.5⇨0.6\n플레어\uF7F6: 9⇨6\n호라이즌\uF7F5: [red]♥[]340⇨270\n제니스\uF7F4: [red]♥[]700⇨580, 16⇨15\n안텀브라\uF7F3: +20%\n이클립스\uF7F2: [red]♥[]20000⇨21000, +25%\n모노\uF7F1: [red]♥[]100⇨70\n폴리\uF7F0: [red]♥[]400⇨280\n리소\uF7E7: [red]♥[]280⇨200, [yellow]\uE86B[]2⇨0, 건물 대상 피해  x1⇨x0.7, 발사체 관성이 사라짐\n민케\uF7ED: 건물 대상 피해  x1⇨x0.7\n브라이드\uF7EC: 80⇨70, 건물 대상 피해  x1⇨x0.7, 발사체 관성이 사라짐",
```

Comment:
I'll accept `BALANCE_` translations for now, but keep in mind they change frequently. Next season I'll have to remove translation to make it actual.